ocaml/idl/datamodel_pool.ml

```diff
@@ -1024,6 +1024,46 @@ let set_uefi_certificates =
       ]
     ~allowed_roles:_R_POOL_ADMIN ()
 
+let update_sync_frequency =
+  Enum
+    ( "update_sync_frequency"
+    , [
+        ("daily", "Indicates the pool update synchronization is scheduled daily")
+      ; ( "weekly"
+        , "Indicates the pool update synchronization is scheduled weekly"
+        )
+      ; ( "monthly"
+        , "Indicates the pool update synchronization is scheduled monthly"
+        )
+      ]
+    )
+
+let configure_update_sync =
+  call ~name:"configure_update_sync"
+    ~doc:"Config scheduled job to sync update from remote CDN" ~lifecycle:[]
+    ~params:
+      [
+        (Ref _pool, "self", "The pool")
+      ; ( update_sync_frequency
+        , "update_sync_frequency"
+        , "The frequency at which updates are synced from remote CDN: daily, \
+           weekly, or monthly."
+        )
+      ; ( Int
+        , "update_sync_day"
+        , "Which day of one period the update sychronization is scheduled. For \
+           'daily' schedule, it should be 1. For 'weekly' schedule, -7..-1 and \
+           1..7, where 1 and -7 is Sunday. For 'monthly' schedule, -28..-1 and \
+           1..28, this ensure that the day exist in every month. For negative \
+           number, it means the day count down from the end of the period."
+        )
+      ; ( Int
+        , "update_sync_hour"
+        , "Which hour of day the update sychronization is scheduled, 0..23"
```

Comment:
So: days in week and in month are 1-based but hours are 0-based.

Reply:
Yes, I set "update_sync_hour" to 0-based, it is the same as the clock hour, and "update_sync_day" for `monthly` to 1-based, it is the same as calendar day, and "update_sync_day" for `weekly` to 1-based to align with `monthly`.

But the week day in Unix is 0-based (https://v2.ocaml.org/api/Unix.html):
tm_wday : int;   Day of week (Sunday is 0)
I am a little hesitate if 0-based is better for day for `weekly`.

What is your suggestion here?